src/main/java/org/ibp/api/java/germplasm/GermplasmService.java

```diff
@@ -59,4 +63,9 @@
 
 	List<org.generationcp.middleware.api.attribute.AttributeDTO> getGermplasmAttributesByCodes(Set<String> codes);
 
+	List<AttributeDTO> getGermplasmAttributes();
```

Comment:
Is it possible to use getGermplasmAttributesByCodes ? I think passing an empty lset will work. We can rename it to filterGermplasmAttributes and we avoid adding another function